### PR TITLE
Fix search pins

### PR DESCRIPTION
**Pull request by falsidge**
## Summary
Search pins were not showing due to undefined offsets. Setting the offset to 0 will make it show up in the proper location. Unfortunately the pin will not stack with class pins.

image

## Test Plan

Searched for place and confirmed popup showed above pin.

## Issues

#304

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "react-lazyload": "^3.1.0",
         "react-leaflet": "^2.7.0",
         "react-router-dom": "^6.2.1",
-        "react-scripts": "^3.4.1",
```
Comment (ChaseC99) [inline package.json]: Do we want to update react-scripts right now?

Comment (falsidge) [reply package.json]: Oh I just realized I have a node-modules folder in a folder above the projects folder, which causes react-scripts to warn about dependencies. We don't need to update it right now/

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Can you revert package-lock.json too plz?

**Review by Douglas-Hong — CHANGES_REQUESTED** [review]
The popups for locations that were searched look pretty good to me! However, the maps for specific days (Mon, Tue, etc.) seem to be broken because of this change. It looks like the pins for each class are one day off.


https://user-images.githubusercontent.com/78244965/160179973-2b40ae9f-db85-4712-9653-b45f163fa0b5.mp4

**Review by Douglas-Hong — APPROVED** [review]
Everything looks good to me!

**Pull request merged**